src/main/java/org/spongepowered/common/event/tracking/UnwindingPhaseContext.java

```diff
@@ -92,4 +92,11 @@ static PhaseContext unwind(IPhaseState state, PhaseContext context) {
         }
         return Optional.ofNullable(super.getSource(sourceClass).orElseGet(() -> this.unwindingContext.getSource(sourceClass).orElse(null)));
     }
+
+    @Override
+    public UnwindingPhaseContext copy() {
+        UnwindingPhaseContext to = new UnwindingPhaseContext(this.unwindingState, this.unwindingContext);
+        copy(to);
+        return to;
+    }
 }
```

Comment:
Fixed.

Just a tip for next time: You can add multiple comments to a single review instead of splitting them among multiple reviews. Just make a comment after you click "Start review" but before you click "Finish your review" or "Submit review".